notebooks/dc2/truth_variable_info.ipynb

```diff
@@ -0,0 +1,520 @@
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "Make sure you have followed the instructions on the [DESC Data Portal](https://lsstdesc-portal.nersc.gov/) to \n",
+    "download the data files, install `GCRCatalogs`, and set up `root_dir` for `GCRCatalogs`.\n",
+    "\n",
+    "In this example notebook, the following files will be needed:\n",
+    "- `lsstdesc-public/dc2/run2.2i-dr6-v4/truth_sn/truth_sn_summary_v1-0-0.parquet`\n",
+    "- `lsstdesc-public/dc2/run2.2i-dr6-v4/truth_sn/truth_sn_variability_v1-0-0.parquet`\n",
+    "- `lsstdesc-public/dc2/run2.2i-dr6-v4/truth_star/truth_star_summary_v1-0-0.parquet`\n",
+    "- `lsstdesc-public/dc2/run2.2i-dr6-v4/truth_star/truth_star_variability_v1-0-0.parquet`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import necessary packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: Remove this cell in final version\n",
+    "import sys\n",
+    "sys.path.insert(0, \"../../../gcr-catalogs\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from astropy.coordinates import SkyCoord"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import GCRCatalogs\n",
+    "from GCRCatalogs import GCRQuery\n",
+    "from GCRCatalogs.utils import first"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def flux2mag(flux):\n",
+    "    \"\"\"\n",
+    "    A convenience function to convert flux to magnitude\n",
+    "    \"\"\"\n",
+    "    with np.errstate(divide=\"ignore\"):\n",
+    "        return 22.5 - 2.5 * np.log10(flux)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: Remove this cell in final version\n",
+    "GCRCatalogs.set_root_dir(\"/global/cfs/cdirs/lsst/gsharing\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Supernova light curves\n",
+    "\n",
+    "We will start with finding light curves for supernovae.  When accessing DC2 variable truth information, one important concept is that the light curves are stored separetely from a \"summary table\". \n",
+    "\n",
+    "In the summary table, each row corresponds to one object (supernova), and it contains \"summary\" truth information such as coordinates, redshift, and time-averaged fluxes, but not the light curves. \n",
```

Comment:
Maybe add a sentence like
"Here _summary_ implies associated with the source generally, not with a particular observation of the source." 
or modify the sentence you have:

In the summary table, each row corresponds to one object (here a supernova), and it contains _summary_ truth information such as coordinates, redshift, and time-averaged fluxes - that is, attributes associated with the object overall, not with a particular observation of it.  Hence it does not include light curve values.